_aggregations/metric/cardinality.md

```diff
@@ -28,22 +44,53 @@ GET opensearch_dashboards_sample_data_ecommerce/_search
 ```
 {% include copy-curl.html %}
 
-#### Example response
+## Example response
+
+The aggregation returns the following:
 
```

Comment:
@dwelsch-esi Ideally, we would provide a little more description re: what the response contains/demonstrates to the user.

Reply:
See suggested change.

Reply:
@dwelsch-esi Looks good, but should "count" follow "cardinality"?

Reply:
And actually, let's flesh out a bit: "As shown in the following example response, the aggregation returns the cardinality count in the `unique_products` variable:"